### PR TITLE
added masking for recommendation url

### PR DESCRIPTION
**Pull request by ppietraszko**
I modified the way that the url for recommendations is displayed. 
Previously it would display as:
http://127.0.0.1:8000/recommendation_rating/<ID OF RECOMMENDATION>/

The id of the recommendation is now encoded making it more difficult to manipulate.

### CODE REVIEW

**Comment by ppietraszko** [comment]
> LGTM. Just one point that would be good to confirm is about the change in the version for django is tested and nothing unexpected happened.

So I did a full test of the site with the django change and nothing appears to be broken.

File: recommendation/tests.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from django.test import TestCase
 from django.urls import reverse
 from django.utils import timezone
+from django.utils.http import urlsafe_base64_encode
```
Comment (snehalkenjale) [inline recommendation/tests.py]: base_64 hash is kind of easy to crack. Is there any better hashing technique that can be used from the same library?

Comment (ppietraszko) [reply recommendation/tests.py]: In that specific library the other one would be base_32 . would you have any recommendations for better hashing?

Comment (snehalkenjale) [reply recommendation/tests.py]: I am not aware of any other library. We can use this for now.

**Pull request merged**